Index: data/ipdbg/PARTS
===================================================================
--- data/ipdbg/PARTS	(revision 0)
+++ data/ipdbg/PARTS	(working copy)
@@ -0,0 +1 @@
+0000111100001111	ipdbg-tap	ipdbg-tap
\ No newline at end of file
Index: data/ipdbg/ipdbg-tap/STEPPINGS
===================================================================
--- data/ipdbg/ipdbg-tap/STEPPINGS	(revision 0)
+++ data/ipdbg/ipdbg-tap/STEPPINGS	(working copy)
@@ -0,0 +1 @@
+1111	ipdbg-tap	1
\ No newline at end of file
Index: data/ipdbg/ipdbg-tap/ipdbg-tap
===================================================================
--- data/ipdbg/ipdbg-tap/ipdbg-tap	(revision 0)
+++ data/ipdbg/ipdbg-tap/ipdbg-tap	(working copy)
@@ -0,0 +1,5 @@
+instruction length 8
+register USER1 12
+register BYPASS 1
+register IDREG 32
+instruction USER1 01010101 USER1
\ No newline at end of file
Index: data/ipdbg/ipdbg-tap/STEPPINGS
===================================================================
--- data/ipdbg/ipdbg-tap/STEPPINGS	(revision 0)
+++ data/ipdbg/ipdbg-tap/STEPPINGS	(working copy)
@@ -0,0 +1 @@
+1111	ipdbg-tap	1
\ No newline at end of file
Index: data/ipdbg/ipdbg-tap/ipdbg-tap
===================================================================
--- data/ipdbg/ipdbg-tap/ipdbg-tap	(revision 0)
+++ data/ipdbg/ipdbg-tap/ipdbg-tap	(working copy)
@@ -0,0 +1,5 @@
+instruction length 8
+register USER1 12
+register BYPASS 1
+register IDREG 32
+instruction USER1 01010101 USER1
\ No newline at end of file
Index: data/ipdbg/ipdbg-tap/ipdbg-tap
===================================================================
--- data/ipdbg/ipdbg-tap/ipdbg-tap	(revision 0)
+++ data/ipdbg/ipdbg-tap/ipdbg-tap	(working copy)
@@ -0,0 +1,5 @@
+instruction length 8
+register USER1 12
+register BYPASS 1
+register IDREG 32
+instruction USER1 01010101 USER1
\ No newline at end of file
Index: data/ipdbg/ipdbg-tap/STEPPINGS
===================================================================
--- data/ipdbg/ipdbg-tap/STEPPINGS	(revision 0)
+++ data/ipdbg/ipdbg-tap/STEPPINGS	(working copy)
@@ -0,0 +1 @@
+1111	ipdbg-tap	1
\ No newline at end of file
Index: data/ipdbg/PARTS
===================================================================
--- data/ipdbg/PARTS	(revision 0)
+++ data/ipdbg/PARTS	(working copy)
@@ -0,0 +1 @@
+0000111100001111	ipdbg-tap	ipdbg-tap
\ No newline at end of file
Index: data/Makefile.am
===================================================================
--- data/Makefile.am	(revision 2052)
+++ data/Makefile.am	(working copy)
@@ -571,5 +571,8 @@
 	xilinx/xc2vp4/STEPPINGS \
 	xilinx/xc2vp4/xc2vp4 \
 	xilinx/xc6slx150t/STEPPINGS \
-	xilinx/xc6slx150t/xc6slx150t
+	xilinx/xc6slx150t/xc6slx150t \
+    ipdbg/PARTS \
+    ipdbg/ipdbg-tap/STEPPINGS \
+    ipdbg/ipdbg-tap/ipdbg-tap
 
Index: data/MANUFACTURERS
===================================================================
--- data/MANUFACTURERS	(revision 2052)
+++ data/MANUFACTURERS	(working copy)
@@ -49,5 +49,6 @@
 00101110000	brecis		Brecis (PMC-Sierra)
 00111101001	marvell		Marvell
 00110101011	marvell		Marvell
+00001111000 ipdbg		ipdbg.org
 # collides with other ARM-based chips
 #11110000111	arm  		ARM
